src/_loadurl.js

```diff
@@ -48,3 +55,18 @@ function getHttpClient(protocol) {
       throw new Error(`unsupported protocol "${protocol}" in url: ${url}`);
   }
 }
+
+function useProxy(hostname) {
+  const hosts = (process.env.no_proxy || '').split(',')
+  for (i in hosts) {
+    host = hosts[i]
+    if (host[0] == '.') {
```

Comment:
Add a comment explaining what's going on here

```suggestion
    if (host[0] === '.') {
```